fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaGenerator.java

```diff
@@ -628,6 +738,67 @@ public void addLogicalResources(PhysicalDataModel pdm) {
         pdm.addObject(tbl);
     }
 
+    /**
+     * Adds a table to support logical identity management of resources
+     * when using a distributed RDBMS such as Citus. It represents the
+     * mapping:
+     * 
+     * (RESOURCE_TYPE_ID, LOGICAL_ID) -> (LOGICAL_RESOURCE_ID)
+     * 
+     * LOGICAL_RESOURCE_ID values are assigned from the sequence FHIR_SEQUENCE.
+     * 
+     * When using Citus (or similar), this table is distributed by LOGICAL_ID,
+     * which means we can use a primary key of {RESOURCE_TYPE_ID, LOGICAL_ID}. 
```

Comment:
a little confused about why we distribute by LOGICAL_ID (e.g. vs using LOGICAL_RESOURCE_ID as both the primary key and the distribution column)

Reply:
maybe just to avoid the implied index since the lookup is always done by `{RESOURCE_TYPE_ID, LOGICAL_ID}`?

Reply:
ok, I guess we also have code that looks up resources by just logical_id (e.g. in JDBCIdentityCacheImpl.java)... I think I get it now